scripts/add_sol_ckusdt_pool_mainnet.sh

```
@@ -0,0 +1,212 @@
+    else
+        print_error "Failed to add SOL token: ${SOL_TOKEN_RESULT}"
+        exit 1
+    fi
+fi
+
+# Step 2: Transfer SOL to Kong's address
+print_header "STEP 2: TRANSFER SOL"
+print_info "Transferring $(echo "scale=9; $SOL_AMOUNT / 1000000000" | bc) SOL to Kong..."
+
+# Transfer SOL
+TRANSFER_OUTPUT=$(solana transfer --allow-unfunded-recipient "$KONG_SOLANA_ADDRESS" $(echo "scale=9; $SOL_AMOUNT / 1000000000" | bc) 2>&1)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)